distribution/docs/src/main/resources/content/_quickstart/quickstart-installing.adoc

```diff
@@ -147,3 +147,53 @@ This installation directory will be referred to as `${home_directory}`.
 ----
 ${branding-lowercase}${at-symbol}local>
 ----
+
+==== Installing Multiple ${branding}s on the Same Host
+
+To have multiple ${branding} instances on the same host, it is necessary to edit the port numbers in the files in the ${branding} install folder.
+
+[cols="5*.^" options="header"]
+|===
+
+|File to Edit
+|Property
+|Original Value
+|Example of New Value
+|Notes
+
+|`bin/inc`
+|`DEFAULT_JAVA_DEBUG_PORT`
```

Comment:
Debug ports only need to be changed if you are running in debug mode which seems like a developer only concern and not needed for the quickstart.

Reply:
This just emphasize the issue at had since this whole section is only for developers and hard to maintain anyway. I am from the opinion that we should remove it.